src/gas-price/gas-price.test.ts

```diff
@@ -53,16 +53,9 @@ describe('gas price', () => {
     beforeEach(() => {
       initializeGasStore(chainId, providerName);
       // Reset the gasPriceStore
-      const state = getState();
-      setState({
-        ...state,
-        gasPriceStore: {
-          ...state.gasPriceStore,
-          [chainId]: {
-            ...state.gasPriceStore[chainId],
-            [providerName]: { gasPrices: [], sponsorLastUpdateTimestampMs: {} },
-          },
-        },
+      updateState((draft) => {
+        draft.gasPriceStore[chainId] = { [providerName]: { gasPrices: [], sponsorLastUpdateTimestampMs: {} } };
+        return draft;
```

Comment:
I don't think we nee to return `draft` because the return value of the produce will return the modified state.